### PR TITLE
jira posttriage: Check the Test automation flag

### PR DESCRIPTION
**Pull request by pierreprinetti**

### CODE REVIEW

File: cmd/posttriage/main.go [hunk 1]
```diff
@@ -55,38 +56,66 @@ func main() {
 			jiraBaseURL,
 		)
 		if err != nil {
-			log.Fatalf("error building a Jira client: %v", err)
+			log.Fatalf("FATAL: error building a Jira client: %v", err)
```
Comment (stephenfin) [inline cmd/posttriage/main.go:59]: nit: Is the `FATAL:` prefix not unnecessary? I'd have expected the `logging` framework to support configuration of prefixes like this?

**Review by stephenfin — COMMENTED** [review]
/lgtm

File: cmd/posttriage/main.go (hunk 1 above)
Comment (pierreprinetti) [reply cmd/posttriage/main.go:59]: yeah probably but I was lazy

Comment (stephenfin) [reply cmd/posttriage/main.go:59]: Fair :joy:

**Pull request merged**